lib/shared/bucketing/__tests__/bucketing.test.ts

```diff
@@ -25,8 +25,8 @@ describe('User Hashing and Bucketing', () => {
             _audience: { _id: 'id', filters: [] } as unknown as Audience,
             _id: 'target',
             distribution: [
-                { _variation: 'var1', percentage: 0.25 },
-                { _variation: 'var2', percentage: 0.45 },
+                { _variation: 'var1', percentage: 0.2555 },
```

Comment:
you should probably update the expected value ranges below.

Reply:
updated to narrow the expected range to be 0.5% around `var1` and `var2`

Reply:
@jonathannorris thoughts on the ranges.. these tests will flake even with 50k evals due to random distributions being, well random 😅

Reply:
@kaushalkapasi has that flakeness changed? it shouldn't be that bad...

Reply:
nope.. with 4 decimal places and the narrowing of the range checks (was previously  +/- 1% of the distribution percentage, now its +/- 0.5%) there's just a higher likely hood of things being just outside that +/- 0.5%

Reply:
the existing tests cover that the config is saved and parsed correctly.. should we add an explicit check for the 4 decimal places? or is the existing test that the config is valid / matches the input config when parsed out of wasm enough?

Reply:
does that test config have these longer decimal places in them?

Reply:
yeah, I've updated the underlying `lib/shared/bucketing-test-data/src/data/testData.ts` file to include a couple of instances to use numbers with 4 decimal places